package.json

```diff
@@ -58,7 +58,6 @@
         "postcss-loader": "^8.1.1",
         "postcss-rem-to-pixel": "^4.1.2",
         "prettier": "^3.2.5",
-        "resolve-url-loader": "5.0.0",
```

Comment:
the bundler plugin resolves all source files in CSS w/o additional source map option